go/vt/tabletserver/config.go

```diff
@@ -128,6 +135,22 @@ var DefaultQsConfig = Config{
 	TwoPCEnable:             false,
 	TwoPCCoordinatorAddress: "",
 	TwoPCAbandonAge:         0,
+
+	EnableTxThrottler: true,
+	// These are mostly the defaults set in the defaultMaxReplicationLagModuleConfig in
+	// go/vt/throttler/max_replication_lag_module_config.go
+	TxThrottlerConfig: `target_replication_lag_sec: 2
```

Comment:
I think proto-spec is not a well-understood format. Should we use the more widely used JSON format instead?